drivers/console/Kconfig

```diff
@@ -207,7 +215,7 @@ config IPM_CONSOLE_THREAD
 
 config IPM_CONSOLE_QUEUE_SIZE
 	int "IPM console queue size"
-	default 4096
+	default 16
```

Comment:
That seems like a pretty impactful size change!  Is this moving from characters to lines or something?  Seems like it needs documentation about what the units are here.